lib/components/app/responsive-webapp.js

```diff
@@ -243,10 +246,13 @@ class RouterWrapperWithAuth0 extends Component {
             <Route exact path={ACCOUNT_PATH}>
               <Redirect to={TRIPS_PATH} />
             </Route>
+            <Route exact path={CREATE_ACCOUNT_PATH}>
+              <Redirect to={CREATE_ACCOUNT_TERMS_PATH} />
```

Comment:
I'd prefer that the Verify screen also get its own path. Not sure how many headaches that would cause though...

Reply:
@landonreed  How about having this route redirect to `account/create/verify`, which would briefly flash and redirect to `account/create/terms` if the email has been verified? (The `Verify my email` would still simply refresh the page.)

Reply:
Addressed the route for the verify screen in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/2df41d8f67089d1a9dd4d77d29975f807b7b7a6b.